packages/components/src/Chip/Chip.mdx

```diff
@@ -0,0 +1,42 @@
+---
+name: Chip
+menu: Components
+route: /components/chip
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { Chip } from ".";
+
+# Chip
+
+```ts
+import { Chip } from "@jobber/components/Chip";
+```
+
+<Playground>
+  <Chip
+    label="Darth Vader"
+    dismissAction={{
+      ariaLabel: "Remove",
+      onDismiss: () => alert("Luke, I am your father"),
+    }}
+  />
+</Playground>
+
```

Comment:
Would like some feedback on what to document here as I'm not sure what considerations were made when it was designed

Reply:
@chris-at-jobber could probably help or point you to the right person/place